modelon/impact/client/entities/case.py

```diff
@@ -216,7 +239,19 @@ def info(self) -> Dict[str, Any]:
     def run_info(self) -> _CaseRunInfo:
         """Case run information"""
         run_info = self._info["run_info"]
-        return _CaseRunInfo(CaseStatus(run_info["status"]), run_info["consistent"])
+        started = (
+            datetime.fromtimestamp(run_info["datetime_started"] / 1e3)
+            if run_info["datetime_started"]
+            else None
+        )
+        finished = (
+            datetime.fromtimestamp(run_info["datetime_finished"] / 1e3)
+            if run_info["datetime_finished"]
+            else None
+        )
```

Comment:
Maybe this can be factored out like:
`finished = _datetime_from_unix_time(run_info.get("datetime_finished"))`
?